src/main/java/mate/academy/dto/CreateBookRequestDto.java

```diff
@@ -1,15 +1,22 @@
 package mate.academy.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Getter;
 import lombok.Setter;
 
 @Getter
 @Setter
 public class CreateBookRequestDto {
+    @NotNull
```

Comment:
use `@NotBlank` to validate String
fix other places
```suggestion
    @NotBlank
```